quetz/tests/test_auth.py

```diff
@@ -439,20 +439,23 @@ def test_private_channels_download(db, client, data, channel_dirs):
     assert response.text == "file content 0"
 
     # fail on private channel without credentials
-    response = client.get('/get/privatechannel/noarch/current_repodata.json')
-    assert response.status_code == 401
+    for method in [client.get, client.head]:
```

Comment:
@jonashaag, what would speak against adding a method callable as a parametrization instead of adding an iteration on each assertion?

Reply:
That was my first approach. The problem is that you need this in very few places only

Reply:
Now I see it. 🙈